src/www/views/servers_view/server_list_item/server_hero_card.ts

```diff
@@ -0,0 +1,223 @@
+/*
+  Copyright 2021 The Outline Authors
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import '@material/mwc-button';
+import '@material/mwc-icon-button';
+import '@material/mwc-menu';
+import '../server_connection_indicator';
+
+import {html, css} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+import {createRef, Ref, ref} from 'lit/directives/ref.js';
+import {Menu} from '@material/mwc-menu';
+
+import {ServerConnectionState} from '../server_connection_indicator';
+import {ServerListItemElement} from '.';
+
+@customElement('server-hero-card')
+export class ServerHeroCard extends ServerListItemElement {
+  @property({attribute: 'root-path'}) rootPath: string;
+  @property() localize: (messageID: string) => string;
+
+  menuRef: Ref<Menu> = createRef();
+
+  static styles = css`
+    /* TODO(daniellacosse): global css reset */
+    h2,
```

Comment:
auto-prefixing and css resetting are things I need to prioritize as we modernize. i'm not sure currently how best to do these things with web components, but i have some ideas.

Reply:
Right, that works for the reset and variables, but doesn't solve the autoprefixing issue.

[All the examples I could find for autoprefixing in lit use rollup.](https://stackoverflow.com/questions/56097519/how-to-use-autoprefixer-with-web-components-litelement)

Reply:
So latest on this - to use these features we'll need to upgrade `postcss`. To upgrade `postcss` we'll need to get rid of our custom RTL plugin. Thankfully, the CSS I'm writing should soon make that possible.